.gitignore

```diff
@@ -29,6 +29,7 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env
```

Comment:
this has already been added below. Location wise this makes more sense. Can you delete the one below?

Reply:
Sure, I'm on it